### PR TITLE
ANDROID-9466 telefonica skin

### PR DESCRIPTION
**Pull request by manolovn**
### :goal_net: What's the goal?
Add Telefonica skin

image
image
image

### ☑️ Checks
- [x] I updated the documentation (readmes, wikis, etc). If no need to update documentation explain why.
- [x] Tested with dark mode.

### :test_tube: How can I test this?
frame (9)

- [x] 🖼️ Screenshots/Videos
- [x] Reviewed by Mistica design team

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -1,119 +1,101 @@
 <component name="ProjectCodeStyleConfiguration">
```
Comment (gmerinojimenez) [inline .idea/codeStyles/Project.xml:1]: watchout! :)

File: library/src/main/res/values/themes_movistar.xml [hunk 2]
```diff
@@ -147,7 +147,7 @@
     </style>
 
     <!-- Just a copy of the above. As themes do not support multiple inheritance. -->
-    <style name="MisticaTheme.Movistar.Prominent">
+    <style name="MisticaTheme.Telefonica.Prominent">
```
Comment (gmerinojimenez) [inline library/src/main/res/values/themes_movistar.xml:150]: intended?

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (manolovn) [reply .idea/codeStyles/Project.xml:1]: F

**Review by gmerinojimenez — APPROVED** [review]

File: library/src/main/res/values/themes_movistar.xml (hunk 2 above)
Comment (manolovn) [reply library/src/main/res/values/themes_movistar.xml:150]: verifying with design team

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (manolovn) [reply .idea/codeStyles/Project.xml]: reverted

File: library/src/main/res/values/themes_telefonica.xml [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        <item name="colorLoginLoadingGradientSecond">?colorBrandDark</item>
+        <item name="colorLoginLoadingGradientThird">?colorBrandDark</item>
+        <item name="colorLoginLoadingGradientFourth">?colorBrandDark</item>
+
+        <!-- Feedbacks -->
+        <item name="feedbackScreenSuccessWithGradient">false</item>
+        <item name="feedbackScreenSuccessInverse">true</item>
+        <item name="feedbackScreenSuccessAnimation">@raw/feedback_success</item>
+        <item name="feedbackScreenErrorAnimation">@raw/feedback_error_movistar</item>
+        <item name="feedbackScreenInfoIcon">@drawable/feedback_info_movistar</item>
```
Comment (JorgeMucientes) [inline library/src/main/res/values/themes_telefonica.xml:132]: hmm, should we duplicate this assets? do they share the same colors between telefonica and Moves?

Comment (manolovn) [reply library/src/main/res/values/themes_telefonica.xml:132]: the colors in that resources are as attributes, so there's no problem.
anyway, to ensure semantic trazability I've duplicated the resources with telefonica specific naming.
nice catch!

**Review by JorgeMucientes — APPROVED** [review]

**Review by JorgeMucientes — APPROVED** [review]

**Review by JorgeMucientes — APPROVED** [review]

File: catalog/src/main/java/com/telefonica/mistica/catalog/ui/fragment/HeadersCatalogFragment.kt [hunk 4]
```diff
@@ -49,25 +52,34 @@ class HeadersCatalogFragment : Fragment() {
...
+            }
             setSubtitle(view.findViewById<TextInput>(R.id.input_subtitle).text.toString())
             setSubtitleHasSecondaryColor(
                 view.findViewById<CheckBoxInput>(R.id.check_subtitle_secondary).isChecked()
             )
+            setSampleBottomLayout(view.findViewById<CheckBoxInput>(R.id.check_bottom_layout).isChecked())
+        }
+    }
+
+    private fun setSampleBottomLayout(checked: Boolean) {
```
Comment (gmerinojimenez) [inline catalog/src/main/java/com/telefonica/mistica/catalog/ui/fragment/HeadersCatalogFragment.kt:78]: this change is related with telefonica skin?

**Review by gmerinojimenez — APPROVED** [review]

**Review by yceballost — CHANGES_REQUESTED** [review]
I saw some wrong colors applications in dark mode, in cards for example is not applying correctly backgroundContainer

can you check if backgroundContainer = "$darkModeGrey" in dark mode?

Controls also has a wrong color in dark mode
checkbox "tick" has to be in white, and switch is not applying correct colors in dark mode

Stepper, progress specifically has a weird color when you click on "next" and the progress is filled, I think is applying MovistarBlue instead of TelefonicaBlue (I think is a problem with the rest of the brands too)

By other side, I detected a wrong color application in tabs in dark mode for Movistar, O2, Vivo, Classic O2, bottom "divider" in tabs is disappeared

You can check color application using mistica storybook to know the correct application (for control and the rest of components)
https://mistica-web.vercel.app/?path=/story/components-forms-switch--default

**Review by yceballost — APPROVED** [review]

**Pull request merged**